src/commands/rover/WhoisCommand.js

```diff
@@ -244,7 +244,7 @@ class WhoisCommand extends Command {
           0: "Visitor",
         }
         
-        if (devforumData !== false) {
+        if (devforumData) {
```

Comment:
wouldn't this technically be interpreted the same as `if (devforumData !== false)`?

Reply:
While undefined is falsy, the difference is that `!== false` checks if it's equal to false, and it's not equal to false as it was never coerced into a boolean, while this change would do that.

Reply:
ah I see